pkg/assembler/clients/operations/package.graphql

```diff
@@ -38,3 +38,49 @@ query Packages($filter: PkgSpec!) {
     ...AllPkgTree
   }
 }
+
+query PackageTypes($filter: PkgSpec!) {
+  packages(pkgSpec: $filter){
+    type
+  }
+}
+
+query PackageNamespaces($filter: PkgSpec!) {
+  packages(pkgSpec: $filter){
+    type
+    namespaces {
+        namespace
+    }
+  }
+}
+
+query PackageNames($filter: PkgSpec!) {
+  packages(pkgSpec: $filter){
+    type
+    namespaces {
+        namespace
+        names {
+            name
+        }
+    }
+  }
+}
+
+query PackageVersions($filter: PkgSpec!) {
```

Comment:
also this could be replaced by the existing `Packages` query if you are ok with the IDs?

Reply:
It seems like the visualizer is having issues resolving `AllPkgTree` and requires these defined

Reply:
as discussed will leave this for now, and resolve the visualizer not understanding the fragments separately